mlflow/environment_variables.py

```diff
@@ -668,7 +668,15 @@ def get(self):
     "MLFLOW_MODEL_ENV_DOWNLOADING_TEMP_DIR", str, None
 )
 
-# Specifies weather to log environment variable names used during model logging.
+# Specifies whether to log environment variable names used during model logging.
 MLFLOW_RECORD_ENV_VARS_IN_MODEL_LOGGING = _BooleanEnvironmentVariable(
     "MLFLOW_RECORD_ENV_VARS_IN_MODEL_LOGGING", True
 )
+
+# Specifies whether to convert a {"messages": [{"role": "...", "content": "..."}]} input
```

Comment:
Should we add a documentation for this env var?

Reply:
https://mlflow.org/docs/latest/python_api/mlflow.environment_variables.html#module-mlflow.environment_variables documents this. But yea we should probably add a code example in langchain

Reply:
Thanks! Yeah, having the documentation in the usecase specific documents should be helpful